### PR TITLE
Msw 91444646 wait period

### PR DESCRIPTION
**Pull request by michaelwehrley**
Add Wait Period

* Add wait condition for users to be able to move to the "Evaluation" stage.
* Add specs for timing functions.
* Update start controller and CurrentUser initialize function.
* Slight refactors.

[Finishes: #91444646]

### CODE REVIEW

File: test/unit/services/CurrentUserSpec.js [hunk 1]
```diff
@@ -89,4 +179,51 @@ describe('CurrentUser', function() {
...
+      expect(CurrentUser.waited()).to.be.false;
+    });
+
+    it('returns true if the user\'s waitPeriod has ended', function() {
+      CurrentUser.setWaitPeriod(-1000);
+
+      expect(CurrentUser.waited()).to.be.true;
+    });
+
+    it('returns true if the user\'s waitPeriod has ended', function() {
```
Comment (ericcf) [inline test/unit/services/CurrentUserSpec.js]: Is this description accurate?

Comment (michaelwehrley) [inline test/unit/services/CurrentUserSpec.js]: Thanks for that, and update should be made.

File: src/js/controllers/PagesController.js [hunk 2]
```diff
@@ -14,16 +13,32 @@
...
+
+    this.displayTimer = function(user) {
+      this.isDisplayable = false;
+      if (user.beforeTimer() || user.waited()) {
+        this.isDisplayable = true;
+      } else if (!user.isWaiting) {
+        var self, stop;
+
+        self = this;
+        user.isWaiting = true;
```
Comment (ericcf) [inline src/js/controllers/PagesController.js]: How will `isWaiting` get reset to `false` for the next participant?

Comment (ericcf) [inline src/js/controllers/PagesController.js]: Nevermind, I see where that happens.

**Pull request merged**